tests/ArrayField_Test.py

```diff
@@ -25,13 +20,12 @@ def test_arrayfield_set_invalid_type_multi_field(self):
         class multi_int_field_packet(models.Packet):
             list_int_field = models.ArrayField(models.IntField(), 10)
 
-        expected_vals = list(range(10))
+        # expected_vals = list(range(10))
```

Comment:
commenting this out isn't necessary.  Go ahead and feel free to remove the line since it's not used at all in the test.